### PR TITLE
Feature/examples/oc halpern

### PR DESCRIPTION
**Pull request by AdrienTaylor**
add tests with examples OC-Halpern & OC-PPM

### CODE REVIEW

**Comment by rkm0959** [comment]
It seems like the tests fail due to precision issues, (I had this problem as well) I think lowering gamma, mu values will help!

**Review by AdrienTaylor — COMMENTED** [review]
Indeed, I tested locally with MOSEK, which is more accurate than SCS :)
-> I lowered the requirements to make them pass with SCS

File: tests/test_examples.py [hunk 1]
```diff
@@ -427,7 +427,7 @@ def test_krasnoselskii_mann_increasing_step_sizes(self):
         self.assertAlmostEqual(wc, ref_pesto_bound, delta=self.relative_precision * ref_pesto_bound)
 
     def test_wc_optimal_contractive_halpern_iteration(self):
-        n, gamma = 7, 2.15
+        n, gamma = 3, .36
```
Comment (rkm0959) [inline tests/test_examples.py:430]: gamma should be >= 1 in OC-Halpern (at least in the paper it is)

Comment (AdrienTaylor) [reply tests/test_examples.py:430]: woops :) thanks!

**Review by bgoujaud — APPROVED** [review]
Perfect for me! Thanks.
I just added a commit with slight formatting adjustments.

**Comment by bgoujaud** [comment]
Feel free to merge Adrien if you are ok with those modifications.

**Pull request merged**